### ISSUE TITLE
Colour contrast is not visible, hard to see where current cursor is on tabs or buttons

### ISSUE THREAD

**Issue by jddcef** [truefedex/tv-bro#203]
**Describe the bug**
I cannot see where the current button is on the bottom bar and on the tab bar as I move around.
The blue colour is too close to white or Gray so it's impossible to see where it is.
Maybe enlarge the button size of the currently selected button so that it's got colour and size, or increase the icon in the button size. Do anything to improve it please.

**To Reproduce**
Try and find the close button, it's not clear where the cursor is.

**Expected behavior**
I expect navigation to be much easier.

**Screenshots**
![17293654154355160270179847383678](https://github.com/user-attachments/assets/7e66f739-b9db-42a1-84a9-8df27d882eec)
I really can't see the blue baby colour buttons.
I've tried it on two different TV models and on a tablet device.

**Environment (please complete the following information):**
 - OS version: Xiaomi TV s

 - any site.
